### PR TITLE
improve styling of collapsing beats

### PR DESCRIPTION
**Pull request by dstonem**
[timeline] toprow icons displaying on hover

[timeline] interstitial icon hovering bugs fixed, including making all icons appear on hover when ui.orientation is vertical

[timeline] vbug fixed: vertical lines no longer collapsing automatically, styling bug fixes to timeline

[timeline] interstitial hovering and makeover implemented on horizontal and vertical

package-lock removed

[timeline] interstitial hovering and makeover implemented on horizontal and vertical

package-lock removed

improve styling of collapsing beats

[timeline] interstitial icon hovering bugs fixed, including making all icons appear on hover when ui.orientation is vertical

[timeline] vbug fixed: vertical lines no longer collapsing automatically, styling bug fixes to timeline

[timeline] interstitial hovering and makeover implemented on horizontal and vertical

package-lock removed

### CODE REVIEW

File: src/app/components/timeline/TimelineTable.js [hunk 1]
```diff
@@ -80,6 +81,23 @@ class TimelineTable extends Component {
     this.props.lineActions.reorderLines(lines, this.props.ui.currentTimeline)
   }
 
+  startHovering = (beat) => {
+<<<<<<< HEAD
```
Comment (Quiescent) [inline src/app/components/timeline/TimelineTable.js]: Looks like you missed a conflict here.
Thankfully it looks like a trivial error.

File: src/app/components/timeline/BeatTitleCell.js [hunk 2]
```diff
@@ -229,6 +231,11 @@ class BeatTitleCell extends PureComponent {
         <Button bsSize={isSmall ? 'small' : undefined} block onClick={this.handleDelete}>
           <Glyphicon glyph="trash" />
         </Button>
+        {hierarchyLevel.level < 2 && (
```
Comment (cameronsutter) [inline src/app/components/timeline/BeatTitleCell.js]: this is what i ended up doing on the horizontal hover options:
`const showExpandCollapse = hierarchyLevels.length - hierarchyLevel.level > 1`
and then in the return:
`showExpandCollapse && ( .....`

File: src/app/components/timeline/CardSVGline.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ export default class CardSVGline extends PureComponent {
   }
 
   render() {
-    console.log('SVG')
+    console.log('SVG ')
```
Comment (cameronsutter) [inline src/app/components/timeline/CardSVGline.js]: Don't forget to remove this

**Review by cameronsutter — COMMENTED** [review]
Just two things, but otherwise looks ready to go

**Pull request merged**